addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -702,3 +480,184 @@ a.no-decoration {
+    float:left;
+    transition: ease .4s;
+    .oe_product_cart {
+        box-shadow: 0 0 20px 0 rgba(0, 0, 0, .02);
+        transition: .4s ease;
+        padding: 0;
+    }
+    form {
+        width: 100%;
+    }
+    .o_product_image {
+        flex: none;
+        height: 44%;
+        position: absolute;
+        top: 3%;
+        left: 10%;
+    }
+    .o_container_product {
+        width: 100%;
+        margin: 0;
+        padding: 0 !important;
+        .o_info_product_container {
+            width: 100%;
+            max-width: 100%;
+            flex: 100%;
+            display: block;
+            padding: 10px 10%;
+            text-align: center;
+            height: 50% !important;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            background-color: $gray-100;
+            transition: .3s ease;
+            z-index: 5;
+            .oe_subdescription {
+                display: none;
+            }
+        }
+        .o_published_btn {
+            top: 0;
+            right: 0;
+            z-index: 9;
+        }
+        small {
+            text-align: center;
+            width: 100%;
+            margin-bottom: 10px;
+        }
+        .o_grid_dropdown {
+            display: inline;
+        }
+    }
+    .o_on_grid {
+        display: block;
+    }
+    .o_not_on_grid {
+        display: none;
+    }
+    .o_btn_product_container {
+        margin-top: 10px;
+        padding-top: 10px;
+        border-top: 1px solid $gray-200;
+    }
+    &:hover {
+        transform: scale(1);
+        z-index: 9;
+        .oe_product_cart {
+            box-shadow: 0 0 20px 0 rgba(0, 0, 0, .02);
+        }
+        .o_container_product .o_info_product_container {
+            height: 100% !important;
+            background-color: $gray-400;
+            .oe_subdescription {
+                display: block;
+                color: $gray-700 !important;
+            }
+            h4 a {
+                color: theme-color('primary') !important;
+            }
+        }
+    }
+}
+.o_grid_products_list.oe_list.oe_product.oe_image_full {
+        .o_product_image {
+            display: none;
+        }
+        .o_product_image_full {
+            height: 100%;
+            max-width: 100%;
+            background-size: cover;
+            background-position: center;
+        }
+        .o_container_product {
+            display: block;
+            height: 100% !important;
+        }
+        form {
+            height: 50%;
+        }
+    }
+.oe_list {
+    transition: ease .4s;
+}
+.oe_list:hover {
+    transform: scale(1.04);
+    z-index: 9;
+    .oe_product_cart {
+        box-shadow: 0 0 40px 0 rgba(0, 0, 0, .2);
+    }
+}
+.nav-pills .nav-item:last-child {
+    border: none !important;
+    padding-bottom: 0 !important;
+    margin-bottom: 0 !important;
+}
+.o_height_line {
+    height: auto !important;
+}
+.o_height_170 {
+    height: 170px !important;
```

Comment:
No... something like `o_big_height` or `o_small_height` I do not know but if a theme chooses it is 450px... it just cannot be called `o_height_170`